viewer/src/main/java/nl/b3p/viewer/userlayer/SQLServer.java

```diff
@@ -20,6 +20,11 @@ public String preValidateView(String tableName, String filterSQL) {
         return null;
     }
 
+    @Override
+    public String getGtPkMetadataDDL() {
+        return null;
```

Comment:
zoiets

```suggestion
        return "CREATE TABLE gt_pk_metadata (table_schema varchar(255), table_name varchar(255), pk_column varchar(255), pk_column_idx int, pk_policy varchar(255), pk_sequence varchar(255))";
```

https://github.com/geotools/geotools/blob/9437e2c6ee3405278b74c154ca3373bb87401a29/modules/plugin/jdbc/jdbc-sqlserver/src/test/java/org/geotools/data/sqlserver/SQLServerPrimaryKeyFinderTestSetup.java#L28-L33

Reply:
Ah, goede. Hij stond niet op de geoserver website.